sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/support/CosmosRepositoryFactory.java

```diff
@@ -54,11 +54,13 @@ public <T, ID> EntityInformation<T, ID> getEntityInformation(Class<T> domainType
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
Do you have plan to remove the usage of deprecated things?

Reply:
maybe new major version or it was removed, no definite plan